README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
```

Comment:
- Your backend API does not have documentation:

![image](https://user-images.githubusercontent.com/57421823/200096858-7a14b7d8-d637-4fce-ae73-e78bafb52e82.png)

You could use [rswag](https://github.com/rswag/rswag).  It's simple to use, and it adds tests and documentation at the same time. 💪